contracts/libraries/Safe150Migration.sol

```diff
@@ -87,22 +86,24 @@ contract Safe150Migration is SafeStorage {
     }
 
     /**
-     * @notice Migrate and set the guard to the specified address.
+     * @notice Migrate and set the guards to the specified address.
```

Comment:
I don't think these migration extensions are needed anymore. In particular, they were there (AFAIU) because 1.5.0 broke existing guards (since the interfaces were combined).

Reply:
https://github.com/safe-global/safe-smart-account/pull/758/commits/781a9d50efa1baabd7a219e0d0d98692dd429741